pkgs/development/python-modules/pyside6/default.nix

```diff
@@ -15,7 +16,7 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "pyside-setup-everywhere-src-${lib.versions.majorMinor version}/sources/${pname}";
 
-  postPatch = ''
+  postPatch = lib.optionalString stdenv.isLinux ''
```

Comment:
Might be worth adding a comment that currently the optional modules are broken on macOS